source/Beacon-GS64-Tests/SignalLoggerTest.class.st

```diff
@@ -0,0 +1,21 @@
+Class {
+	#name : #SignalLoggerTest,
+	#superclass : #TestCase,
+	#category : #'Beacon-GS64-Tests'
+}
+
+{ #category : #tests }
+SignalLoggerTest >> testStartingTwiceDoesNotRegisterTwice [
+	| initialNumberOfSubscriptions |
+	initialNumberOfSubscriptions := Beacon instance announcer numberOfSubscriptions.
+	SignalLogger resetInstance.
+	SignalLogger start.
+	SignalLogger start.
+	self
+		assert: Beacon instance announcer numberOfSubscriptions
+		equals: initialNumberOfSubscriptions + 1.
+	SignalLogger stop.
```

Comment:
We can put `SignalLogger stop` inside an `ensure:` block, so if something fails before it will stop the logger anyway

Reply:
This is Beacon code taken from Pharo. I would keep it as is, to ensure we stick to minimum changes to original code, unless of course we find a problem that needs to be fixed.